corehq/apps/custom_data_fields/edit_model.py

```diff
@@ -104,7 +104,7 @@ def get_validator(cls, domain):
 
     @classmethod
     def page_name(cls):
-        return _("Edit {} Fields").format(cls.entity_string)
+        return _("Edit {} Fields").format(unicode(cls.entity_string))
```

Comment:
Oh. sorry. I meant to update this as well. It's not actually required, but I thought it was at the time because I originally tested it in ipdb with ugettext_lazy imported as _. ipdb actually overwrites _ after every command so I was confused when I made that commit.